addon/background.js

```diff
@@ -83,6 +83,9 @@ async function getTabInfo(tabIds, {wantsScreenshots, wantsReadability}) {
         file: "capture-data.js",
       });
       let data = await browser.tabs.sendMessage(tabId, {type: "getData", wantsScreenshots, wantsReadability});
+      if (data.readability && data.readability.content) {
+        data.readability.content = DOMPurify.sanitize(data.readability.content);
```

Comment:
This looks good!
You could also leave this as is and do the sanitize() near the sink, i.e., the innerHTML assignments. That would allow you to whitelist the `DOMPurify.sanitize()` function in your no-unsanitized linter config and remove the "disable for this line" comments.